src/OrbitCore/Path.cpp

```diff
@@ -33,9 +34,17 @@ static std::string GetEnvVar(const char* variable_name) {
   return var;
 }
 
+static void CreateDirectoryOrDie(const std::filesystem::path& directory) {
+  auto create_directory_result = orbit_base::CreateDirectory(directory);
+  if (create_directory_result.has_error()) {
+    FATAL("Unable to create directory \"%s\": %s", directory.string(),
```

Comment:
Are we fine with just crashing in all the functions below?

Reply:
we have done it before just implicitly, but I agree there is a better way to do it, we will need to separate create and get have create return ErrorMessageOr and call initialization from somewhere we can display the error. Orbit should still probably not start but at least user will know why and it won't be a crash.

But this is for another PR, I did not want to do it now.